src/asciidoc/docs/develop/addService.adoc

```diff
@@ -193,54 +189,126 @@ app.listen(process.env.PORT || 3333, null, async err => {
 +
 This code will start a basic express server, which will respond to a GET with JSON that you can verify on the frontend.
 
+== Accessing internal Koji variables
+
+There are several values your app will need to get from Koji.
+
+* The URL for the frontend.
+* The URL for the backend.
+* The project ID
+
+Koji stores these values in internal variables. To make them accessible to your app, you need to map them to environment variables, and then add them to the configuration.
+
+. Open the frontend/.env file, update the contents as follows, and then save the file.
++
+[source,text]
+----
+REACT_APP_FRONTEND_URL=$KOJI_SERVICE_URL_frontend
+REACT_APP_BACKEND_URL=$KOJI_SERVICE_URL_backend
+REACT_APP_PROJECT_ID=$KOJI_PROJECT_ID
+----
+. Open the frontend/src/index.js file, update the config call as follows, and then save the file.
++
+[source,javascript]
+----
+....
+Koji.config(kojiConfig, {
+  projectId: process.env.REACT_APP_PROJECT_ID,
+  services: {
+    backend: process.env.REACT_APP_BACKEND_URL,
+    frontend: process.env.REACT_APP_FRONTEND_URL
+  },
+});
+....
+----
+
 == Letting Koji know about the backend service
 
-You have now added a backend project, but the Koji editor and project builder still don't know it exists.
-You must update the `develop.json` and `deploy.json` files to let Koji know about the new service.
-Both files are located in the `.koji` folder of the project folder.
+You have now added a backend project, but the Koji editor and app builder still don't know it exists.
+You must update the `koji.json` file to let Koji know about the new service.
+`koji.json` is located in the root directory.
 
-. Open the `.koji/project/develop.json` file and update it to look like the following code.
+. Open the `koji.json` file, and replace the existing content with the following code.
 +
 [source,json]
 ----
-include::../_includes/develop.json[tags=**,indent=0]
+{
+  "deploy": {
+    "frontend": {
+      "output": "frontend/build",
+      "type": "static",
+      "commands": [
+        "cd frontend",
+        "npm install",
+        "export NODE_ENV=production && npm run build"
+      ]
+    },
+    "backend": {
+      "output": "backend",
+      "type": "dynamic",
+      "commands": [
+        "cd backend",
+        "npm install",
+        "export NODE_ENV=production && npm run compile"
+      ]
+    }
+  },
+  "develop": {
+    "frontend": {
+      "path": "frontend",
+      "port": 8080,
+      "startCommand": "npm start"
+    },
+    "backend": {
+      "path": "backend",
+      "port": 3333,
+      "startCommand": "npm run start-dev"
+    }
+  },
+  "entitlements": {
+    "InstantRemixing": true,
+    "InstantRemixingNativeNavigation": true,
+    "FeedEvents": true
```

Comment:
Remove `FeedEvents`. This entitlement was deprecated.

Reply:
This is part of the scaffold. It would have to be removed from there to remove it from this app, unless you wanted to direct the reader to remove it. Otherwise, no reference to it will appear in the doc, since I'm using the include file instead of listing the entire contents of koji.json.

Reply:
This is part of the scaffold. It would have to be removed from there to remove it from this app, unless you wanted to direct the reader to remove it. Otherwise, no reference to it will appear in the doc, since I'm using the include file instead of listing the entire contents of koji.json.

Reply:
No, we shouldn't need to remove it from the scaffold. It shouldn't break anything (it just won't do anything). We just don't want it in the docs. It sounds like this won't be an issue since you made the previous change. Thanks!